### ISSUE TITLE
Move fixtures to OCFL?

### ISSUE THREAD

**Issue by ahankinson** [zimeon/ocfl-fixtures#1]
Would it be possible to move these fixtures to the main OCFL project space?

**Comment by zimeon** [comment]
Yes, we should do that at some time. Perhaps needs to discuss in editors' call how to arrange and to manage.

**Comment by ahankinson** [comment]
Sure; I can add that to the agenda for next week
